ouroboros/chainsync/era.go

```diff
@@ -0,0 +1,72 @@
+package chainsync
```

Comment:
just a collection of helper functions that I think we could utilize instead of doing the check of
`Alonzo != nil || Babbage != nil`

Reply:
Woo hoo!